src/mod/asr_tts/mod_unimrcp/mod_unimrcp.c

```diff
@@ -1961,7 +1961,9 @@ static apt_bool_t speech_on_channel_add(mrcp_application_t *application, mrcp_se
 	return TRUE;
 
 error:
-	if (schannel) {
+	if (schannel->channel_destroyed) {
```

Comment:
We need to check if schannel is not a NULL pointer. Please add this statement inside the `if(schannel)`

Reply:
Added